client/app/styles/index.styl

```diff
@@ -0,0 +1,111 @@
+@import '../../node_modules/normalize.css/normalize.css'
+@import './base/_normalize'
+@import './base/_colors'
+
+@import 'cozy-ui'
+
+[role=application]
+    @extend $app-2panes-toolbar
+    background-color: $grey-01-alpha
+
+    main
+        padding: 2.5em 3em
+
+    h1
+        font-size: 2em
+        margin: 0 0 1em
+
+
+aside
+    background-color: $grey-01
+    box-shadow: inset -1px 0 0 0 $grey-01-alpha
+
+    h4
+        font-weight: normal
+        font-size: 1.5em
+        padding: 1em 1.5em
+        margin: 0
+
+[role="navigation"]
+    li
+        display: flex
+        flex-direction: row
+
+    a
+        text-decoration: none
+        color: $red
+        flex: 1
+        padding: 1em 1.5em
+        margin: 0.25em 0
+
+        &:hover:not(.router-link-active)
+            background-color: $grey-01-alpha
+
+    .router-link-active
```

Comment:
I tried your branch and I figured out that this style is not applied anymore, we don't have the red highlight in the menu anymore. That part was specific to the previous [vue-router](https://router.vuejs.org/en/api/router-link.html). However, the `preact-router` seems to not work like the react-router with a `activeClassName` attribute.